test/Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests/TrxLoggerTests.cs

```diff
@@ -634,6 +639,95 @@ public void DefaultTrxFileShouldCreateIfLogFileNameParameterNotPassed()
             Assert.IsFalse(string.IsNullOrWhiteSpace(this.testableTrxLogger.trxFile));
         }
 
+        [TestMethod]
+        public void DefaultTrxFileNameVerification()
+        {
+            this.parameters.Remove(TrxLoggerConstants.LogFileNameKey);
+            this.parameters[TrxLoggerConstants.LogFilePrefixKey] = DefaultLogFilePrefixParameterValue;
+
+            var time = DateTime.Now;
+            var internalFileHelper = new InternalFileHelper(() => time);
+
+            testableTrxLogger = new TestableTrxLogger(new FileHelper(), internalFileHelper);
+            testableTrxLogger.Initialize(this.events.Object, this.parameters);
+
+            MakeTestRunComplete();
+
+            var fileName = Path.GetFileName(testableTrxLogger.trxFile);
+            var expectedName = $"{DefaultLogFilePrefixParameterValue}{time:_yyyyMMddHHmmss}.trx";
+
+            Assert.AreEqual(expectedName, fileName, "Trx file name pattern has changed. It should be in the form of prefix_yyyyMMddHHmmss.trx");
```

Comment:
Please add the reason why we need it to be this format, which is that Azure Devops VSTest task depends on this naming.